lib/createSentryService.ts

```diff
@@ -0,0 +1,13 @@
+import { SentryModuleOptions } from './sentry.interfaces';
+import { SentryService } from './sentry.service';
+import { ConsoleLogger } from '@nestjs/common';
+
+export function createSentryService(options: SentryModuleOptions): SentryService {
```

Comment:
I would move this to the constructor of `SentryService`, so the defaulting is done there. That makes it easier in general to use this service I think, without having to rely on this factory.

Reply:
Actually, the users of that lib should not have an access to that low level implementation. All what other devs have to do - use `@InjectSentry()` decorator in their projects without caring about what happenes under the hood.

But yeah, maybe for internal usage it makes sense to re-think the instantiation of SentryService.

Reply:
Oh, I see, you wanted to move default logger instantiation to `SentryService`'s constructor! Yep, makes sense!

Reply:
Fixed!